### PR TITLE
New overload for Player::Heal

### PR DESCRIPTION
**Pull request by Ficus-x**

### CODE REVIEW

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -1912,6 +1897,36 @@ public void Heal(float amount, bool overrideMaxHealth = false)
                 Health += amount;
         }
 
+        /// <summary>
+        /// Heals the player.
```
Comment (Thundermaker300) [inline Exiled.API/Features/Player.cs:1901]: "Heals the player by applying an item effect."?

**Review by Thundermaker300 — APPROVED** [review]

File: Exiled.API/Features/Player.cs [hunk 2]
```diff
@@ -14,49 +14,34 @@ namespace Exiled.API.Features
...
     using PlayableScps;
-
     using PlayerStatsSystem;
-
     using RemoteAdmin;
-
     using RoundRestarting;
-
     using UnityEngine;
-
```
Comment (PintTheDragon) [inline Exiled.API/Features/Player.cs:59]: Please don't change the using style.

Comment (Ficus-x) [reply Exiled.API/Features/Player.cs]: Solved

**Comment by Thundermaker300** [comment]
My only concern with this is Northwood changing item effect = method being outdated. I would recommend closing this, as #1355 does the same thing (+ with more items) and accounts for northwood code changes.

**Pull request closed without merging**